tests/common/toolboxtest.cpp

```diff
@@ -0,0 +1,75 @@
+/*
+ * LibrePCB - Professional EDA for everyone!
+ * Copyright (C) 2017 LibrePCB Developers, see AUTHORS.md for contributors.
+ * http://librepcb.org/
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*****************************************************************************************
+ *  Includes
+ ****************************************************************************************/
+
+#include <QtCore>
+#include <gtest/gtest.h>
+#include <librepcb/common/toolbox.h>
+
+/*****************************************************************************************
+ *  Namespace
+ ****************************************************************************************/
+namespace librepcb {
+namespace tests {
+
+/*****************************************************************************************
+ *  Test Class
+ ****************************************************************************************/
+
+class ToolboxTest : public ::testing::Test
+{
+};
+
+/*****************************************************************************************
+ *  Test Methods
+ ****************************************************************************************/
+
+TEST(ToolboxTest, testNumeric)
```

Comment:
Unfortunately in the test output one can't see that this test is related to the method `stringOrNumberToQVariant`:

```
[----------] 3 tests from ToolboxTest
[ RUN      ] ToolboxTest.testNumeric
[       OK ] ToolboxTest.testNumeric (0 ms)
[ RUN      ] ToolboxTest.testNonNumeric
[       OK ] ToolboxTest.testNonNumeric (0 ms)
[ RUN      ] ToolboxTest.testMixed
[       OK ] ToolboxTest.testMixed (0 ms)
[----------] 3 tests from ToolboxTest (0 ms total)
```

I see different ways how to fix that:
1. Change the name of the test cases to:
  - `testStringOrNumberToQVariant_numeric`
  - `testStringOrNumberToQVariant_nonNumeric`
  - `testStringOrNumberToQVariant_mixed`
2. Put all three tests into one test with the name `testStringOrNumberToQVariant`
3. Rename the whole test (and its file) to `ToolboxStringOrNumberToQVariantTest`